CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [1.1.0] - TBD
+
+### Added
+
+#### CivisML
+- New named workflows: `multilayer_perceptron_regressor`, `multilayer_perceptron_classifier`,
+`stacking_regressor`,  and `stacking_classifier`
+- Hyperband is provided for hyperparameter tuning by setting \code{cross_validation_parameters = "hyperband} in `gradient_boosting_classifier`, `random_forest_classifier`, `extra_trees_classifier`, `multilayer_perceptron_classifier`, `gradient_boosting_regressor`, `random_forest_regressor`, `extra_trees_regressor`, and `multilayer_perceptron_regressor`.
+- Hyperparameter tuning by grid search is now distributed across EC2 instances by setting `n_jobs > 1`
```

Comment:
All hyperparameter tuning is distributed now (both grid search and hyperband). This is probably something I need to clarify in the civis-python docs, actually...